.github/workflows/update-xla-commit-hash.yml

```diff
@@ -20,38 +20,59 @@ jobs:
         with:
           fetch-depth: 1
           submodules: false
+          token: ${{ secrets.PAT }}
```

Comment:
Did you already add this secret to our repo?

Reply:
oops